dashboard/app/views/levels/editors/_weblab.html.haml

```diff
@@ -11,3 +18,10 @@
     %li autocreate a project that is shared with all levels with the same project template level
   %p (Leave blank if you do not want all of this)
   = f.text_field :project_template_level_name, placeholder: 'name of level you want to use as the project template'
+
+.field.teacher-only-markdown
+  %p Edit Teacher Only Markdown:
+  = f.text_area 'teacher_markdown', placeholder: 'Enter teacher only markdown here', rows: 4, value: @level.teacher_markdown or ''
+  #level_teacher_markdown_preview
+  :javascript
```

Comment:
More generally, could extract this into its own partial like [_md_instructions.haml](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/_md_instructions.haml) or [_encrypted_examples.html.haml](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/_encrypted_examples.html.haml).